components/core/file_sender/sender.py

```diff
@@ -5,20 +5,24 @@
 
 @server.route("/api/get/file", methods=['GET'])
```

Comment:
REST of our API endpoints are in `REST.py`. How about we define the endpoint there and have the internals here?

Reply:
Ya, it seems a good way. I guess blueprint module can help us in this way [here](http://flask.pocoo.org/docs/1.0/blueprints/)

Reply:
I wonder if we really need Blueprints as the API is simple and have just few endpoints.

Reply:
Either, we can add a task of restructuring the APIs in a logical way. Else the APIs are already defined in a good way and structure of project is also good

Reply:
So I guess we can merge this PR? @agentmilindu

Reply:
@malithsen Please give your comments on this.

Reply:
Since our API is simple enough, let's just define the endpoints in REST.py and have the functionality implemented in modules. As for this PR let's move the highlighted line to REST.py and keep the internals here

Reply:
I guess this is what we can use 
http://flask.pocoo.org/docs/0.12/api/#flask.Flask.add_url_rule

Reply:
I have changed the code, please review it back @malithsen 
Also please suggest better ways to do this 😊